CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Nothing yet!
+### Fixed
+
+- Error when `layer(…)` in `@import` is not first in the list of functions/conditions ([#15109](https://github.com/tailwindlabs/tailwindcss/pull/15109))
```

Comment:
```suggestion
- Error when `layer(…)` as an argument to the `@import` rule is not on first place ([#15109](https://github.com/tailwindlabs/tailwindcss/pull/15109))
```

Not sure if this is really clearer though :/  Take this as an idea only lol

Reply:
This was already an updated version from @adamwathan haha

See: https://github.com/tailwindlabs/tailwindcss/pull/15109#discussion_r1854128092

Reply:
😬 in that case, nothing to see here